.github/workflows/phpunit.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             phpVersion: "8.4"
           - name: service-pdf
             path: "./service-pdf"
-            minCoverage: 85
+            minCoverage: 84
```

Comment:
Neccessary evil - the way SqsWorker is written is going to make testing tricky in its current form. We should probably consider if we want to do a small rewrite to start injecting some services rather than instantiating mid-function.